modulus/experimental/resdiff/training/img_utils.py

```diff
@@ -101,12 +106,9 @@ def reshape_fields(
                 )
             img = np.concatenate((img, grid), axis=1)
 
-    if params.roll:
+    if roll:
         img = np.roll(img, y_roll, axis=-1)
 
-    # if train and (crop_size_x or crop_size_y):
-    #     img = img[:,:,rnd_x:rnd_x+crop_size_x, rnd_y:rnd_y+crop_size_y]
-
     if crop_size_x or crop_size_y:
```

Comment:
To me, removing `train` condition (as in the commented out code) is a bit strange - do they want to apply random crops during evaluation too?

Reply:
Yeah I'm not sure what the expected behavior is for the evaluation. I will add a TODO comment here to have an eye on it.